collectors/osd.go

```diff
@@ -500,6 +533,79 @@ func (o *OSDCollector) collectOSDDump() error {
 
 }
 
+func (o *OSDCollector) collectOSDTree() error {
```

Comment:
Would it be possible to throw a quick test for `collectOSDTree()`? There are examples in `osd_test.go` which should help provide a sample.